lib/metadata/virtual_container_host.go

```diff
@@ -56,73 +56,53 @@ type VirtualContainerHostConfigSpec struct {
 	InfrastructureAdmin []mail.Address
 
 	// Certificates for user authentication - this needs to be expanded to allow for directory server auth
-	UserCertificates []tls.Certificate
+	UserCertificates []*RawCertificate
 	// Certificates for general outgoing network access, keyed by CIDR (IPNet.String())
-	NetworkCertificates map[string]tls.Certificate
+	NetworkCertificates map[string]*RawCertificate
+	// The certificate used to validate the appliance to clients
+	HostCertificate *RawCertificate `vic:"0.1" scope:"read-only"`
+	// The CAs to validate client connections
+	CertificateAuthorities []byte `vic:"0.1" scope:"read-only"`
 	// Certificates for specific system access, keyed by FQDN
-	HostCertificates map[string]tls.Certificate
+	HostCertificates map[string]*RawCertificate
 
 	// Port Layer - storage
-	// Datastore used for the ImageStore
-	// TODO this will change, probably to url.URL, as needed by the port layer
-	ImageStore string `vic:"0.1" scope:"read-only" key:"image_store"`
+	// Datastore URLs for image stores - the top layer is [0], the bottom layer is [len-1]
+	ImageStores []url.URL `vic:"0.1" scope:"read-only" key:"image_stores"`
```

Comment:
post-GA most likely - I moved this to an array because where we've just had string values there's assumptions being made that the value is a) simple, b) always going to be a single value.
Using an array is a clear indicator that we expect to have more than one value for this in the future.